src/components/Reusable/Card.js

```diff
@@ -0,0 +1,16 @@
+import { useTheme } from "../../contexts/theme.context"
+
+const Card = (props) => {
+    const { theme } = useTheme()
+
+    return (
+
+        <article className={`${props.classes} ${theme === 'dark' && 'b-solid b-secondary'} card-dim card-shadow-xs`}>
```

Comment:
good use of conditional class names here.

Reply:
thank you.